File: python/SSAFY 입과전/Easy02.java
import java.util.Scanner;

public class Easy02 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in); 
        int num = scan.nextInt(); 
        int answer = 0; 

        while(num > 0) {
            int one = num % 10; 
            answer += one; 
            num = num / 10;
        }

        System.out.println(answer);
    }
}

File: python/SSAFY 입과전/easy02.py
'''
하나의 자연수를 입력받아 각 자릿수의 합을 계산하는 프로그램 

'''

num = int(input())
answer = 0

while num > 0: 
    one = num % 10
    answer += one
    num = num // 10

print(answer)